src/common

```diff
@@ -150,6 +150,64 @@ else
   $(error Invalid setting for BUILD_ENV: $(BUILD_ENV_))
 endif
 
+# select default solver to be minisat2 if no other is specified
+ifndef IPASIR
+ifndef PRECOSAT
+ifndef PICOSAT
+ifndef LINGELING
+ifndef CHAFF
+ifndef BOOLEFORCE
+ifndef MINISAT
+ifndef MINISAT2
+ifndef GLUCOSE
+MINISAT2 = ../../minisat-2.2.1
```

Comment:
This will be the only path in common -- a bit of a deviation from the current scheme.
I see you want to be able to switch solvers by giving a command line option to make.
Can that be achieved in any other way?

Reply:
Shouldn't we just fail if no solver has been set? No need for defaults, the user should be forced to choose at least one. I'd do
````make
ifeq ($(BOOLEFORCE)$(CHAFF)$(GLUCOSE)$(IPASIR)$(LINGELING)$(MINISAT)$(MINISAT2)$(PICOSAT)$(PRECOSAT),)
  $(error Please select a SAT solver)
endif
````